requirements/default.txt

```diff
@@ -8,9 +8,11 @@ dockerflow==2019.6.0 \
 everett==1.0.2 \
     --hash=sha256:66c7991c056fd9b1de54c29e2b5004a30209530faef698d6ecfbf24bfbbdd1fb \
     --hash=sha256:8a5ee8cdde0ed6b3bbb63249e0e0703d4ad91921107ff198e0868ffce5f7b6bc
+# NOTE(willkg): Falcon 2.0 changes a bunch of things, so there's some non-trivial
+# rewrite work to upgrade to Falcon 2+.
 falcon==1.4.1 \
     --hash=sha256:0a66b33458fab9c1e400a9be1a68056abda178eb02a8cb4b8f795e9df20b053b \
-    --hash=sha256:3981f609c0358a9fcdb25b0e7fab3d9e23019356fb429c635ce4133135ae1bc4
+    --hash=sha256:3981f609c0358a9fcdb25b0e7fab3d9e23019356fb429c635ce4133135ae1bc4 # pyup: <2.0.0
```

Comment:
This requires some non-trivial work. I forget what they changed specifically, but I remember looking at it and thinking it was a fair bit of work because of some of the ways we do things in Antenna. In the meantime, I tossed in an "ignore this" so pyup stops trying.